examples/skr/README.md

```diff
@@ -131,27 +106,27 @@ A fake encryption key is used in the command below to see the key get released.
 go run /tools/importkey/main.go -c importkeyconfig.json -kh /path/to/encryptionKeyFile
 ```
 
-Upon successful import completion, you should see something similar to the following: 
+Upon successful import completion, you should see something similar to the following:
 
 ```
 [34 71 33 117 113 25 191 84 199 236 137 166 201 103 83 20 203 233 66 236 121 110 223 2 122 99 106 20 22 212 49 224]
-https://accmhsm.managedhsm.azure.net/keys/doc-sample-key-release/8659****0cdff08
-{"version":"1.0.0","anyOf":[{"authority":"https://sharedeus2.eus2.test.attest.azure.net","allOf":[{"claim":"x-ms-sevsnpvm-hostdata","equals":"aaa7***7cc09d"},{"claim":"x-ms-compliance-status","equals":"azure-compliant-uvm"},{"claim":"x-ms-sevsnpvm-is-debuggable","equals":"false"}]}]}
+https://<mhsm-name>.managedhsm.azure.net/keys/doc-sample-key-release/8659****0cdff08
+{"version":"1.0.0","anyOf":[{"authority":"<authority-url-name>","allOf":[{"claim":"x-ms-sevsnpvm-hostdata","equals":"aaa7***7cc09d"},{"claim":"x-ms-compliance-status","equals":"azure-compliant-uvm"},{"claim":"x-ms-sevsnpvm-is-debuggable","equals":"false"}]}]}

```

Comment:
I think this authority is pretty stable for people in eastus.  We should at least provide a link to the doc on where to find these.

Reply:
Info on the authority is found in the readme when telling people how to create an MAA instance. We just didn't want our test MAA/AKV names in the documentation